### PR TITLE
Add Running Tests through Stoplight article

### LINKED ISSUE #104: articles/testing/running-tests-stoplight.md

### PR DESCRIPTION
**Pull request by rossmcdonald**
Fixes #104

### CODE REVIEW

File: articles/testing/run-test-stoplight.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+* **Collections**: a series of **scenarios** that encapsulate an
+  entire test suite. Collections are the highest-level building blocks for creating
+  a library of API interactions and tests.
+
+Each level above can be run individually or all together.
+
+## Running a Step
+
+Once you have created a scenario **step**, you can run a single step by using
+the **Run Step** button available towards the top of the editor while viewing
```
Comment (rowa97) [inline articles/testing/run-test-stoplight.md]: numbered list could help here, creating these complex sentences with lots of supplementary information becomes hard to read

File: articles/testing/run-test-stoplight.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+![](../../assets/images/run-test-stoplight2.png)
+
+<!-- theme: info -->
+
+> Scenarios can also be run directly from every step using the **Run Scenario**
+> button
+
+## Running a Collection
+
+Once you have added created enough scenarios to compose a **collection**, you
```
Comment (rowa97) [inline articles/testing/run-test-stoplight.md]: same as above

File: articles/testing/run-test-stoplight.md (hunk 1 above)
Comment (rossmcdonald) [reply articles/testing/run-test-stoplight.md]: I broke up and simplified the sentences a little bit. Let me know if the latest changes are still confusing.

File: articles/testing/run-test-stoplight.md (hunk 2 above)
Comment (rossmcdonald) [reply articles/testing/run-test-stoplight.md]: Let me know if the latest version is any better.

**Comment by rossmcdonald** [comment]
@rowa97 Am I good to merge this one, or is there anything else that needs fixing?

**Pull request merged**